examples/misc/test/samples_test.dart

```diff
@@ -23,16 +23,6 @@ void main() {
   const oneSecond = Duration(seconds: 0);
   final someDate = DateTime(1999);
 
-  test('hello world', () {
```

Comment:
Why remove this?

Reply:
Not sure that I remember any good reasoning. I've added it back!